node/src/service.rs

```diff
@@ -641,10 +644,22 @@ async fn start_sh_services(
 
     let blockchain_service_handle = spawn_blockchain_service(&task_spawner).await;
 
-    let sh_handler = StorageHubHandler::new(
+    let file_storage = Arc::new(RwLock::new(InMemoryFileStorage::new()));
```

Comment:
Bare in mind that these function is removed in the Blockchain Service branch, since initialisation of StorageHub services has to be split in two. Anyway, it can be handled in that branch, since most likely this PR will be merged first. But just remember to keep an eye on this.